WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -87,15 +87,13 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "RoundExpiryTimeout", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public TimeSpan RoundExpiryTimeout { get; set; } = TimeSpan.FromMinutes(5);
 
-	[DefaultValue(100)]
+	[DefaultValue(400)]
 	[JsonProperty(PropertyName = "MaxInputCountByRound", DefaultValueHandling = DefaultValueHandling.Populate)]
-	public int MaxInputCountByRound { get; set; } = 100;
+	public int MaxInputCountByRound { get; set; } = 400;
 
-	[DefaultValue(0.5)]
-	[JsonProperty(PropertyName = "MinInputCountByRoundMultiplier", DefaultValueHandling = DefaultValueHandling.Populate)]
-	public double MinInputCountByRoundMultiplier { get; set; } = 0.5;
-
-	public int MinInputCountByRound => Math.Max(1, (int)(MaxInputCountByRound * MinInputCountByRoundMultiplier));
+	[DefaultValue(150)]
+	[JsonProperty(PropertyName = "MinUniqueInputCountByRound", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public int MinUniqueInputCountByRound { get; set; } = 150;
```

Comment:
Maybe this change should be done in a separate PR, since it can be a breaking change for the affiliation servers.